### PR TITLE
More detailed computer vision profile documentation.

### PR DESCRIPTION
**Pull request by ipuustin**

### CODE REVIEW

**Comment by ipuustin** [comment]
@rojkov

File: doc/computervision.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+We especially aim to:
+
+#. Create demos which provide starting points for product development.
+#. Make sure the different components are scriptable with the same
+   scripting language, making it possible to create demos utilizing
+   different computer vision techniques. We have selected Python 3 to be
+   this language. NumPy matrixes are the common format for carrying
+   image data between different components.
+#. Provide hardware acceleration using OpenCL and Intel-specific
+   features for the componets which support it.
```
Comment (pohly) [inline doc/computervision.rst]: typo: components

File: doc/computervision.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+#. Select components thoughtfully, keeping in mind the licensing and
+   security history. Computer vision production configuration doesn't
+   depend on any components which use (L)GPLv3 license, making it easier
+   to employ technologies such as secure boot.
```
Comment (pohly) [inline doc/computervision.rst]: I think the established convention is to write "Secure Boot" (capital letters).

**Comment by ipuustin** [comment]
@pohly: Thanks for the comments! Fixed the issues you mentioned and also some others. :-)

File: doc/computervision.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+==============
+
+`OpenCV <http://opencv.org/>`_ is the backbone of the open source computer
+vision software components. We integrate it with some extra configuraton. We add
+`gstreamer-vaapi <https://gstreamer.freedesktop.org/modules/gstreamer-vaapi.html>`_
+for Intel-accelerated video encoding, decoding, and processing. Intel RealSense
+camera support is provided by
+`librealsense <https://github.com/IntelRealSense/librealsense>`_.
+
+In addition to this, we add OpenCl hardware acceleration capabilities with
```
Comment (gvancuts) [inline doc/computervision.rst]: s/OpenCl/OpenCL

File: doc/computervision.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+`Beignet <https://www.freedesktop.org/wiki/Software/Beignet/>`_. It can be
+accessed using `ViennaCL lineal algebra library <http://viennacl.sourceforge.net/>`_.
```
Comment (gvancuts) [inline doc/computervision.rst]: Is my assumption correct that OpenCV also uses OpenCL by default in the IoT RefKit?

Comment (ipuustin) [inline doc/computervision.rst]: Depends on the way the OpenCV application is written. It needs to use the so-called "Transparent API" (TAPI) calls. In Python scripts you will need to specify that OpenCL should be used.

Comment (gvancuts) [inline doc/computervision.rst]: This is good info, thanks @ipuustin

**Comment by ipuustin** [comment]
Rebased.

**Pull request merged**